changelog.md

```diff
@@ -3,6 +3,7 @@
 ### 1.9.5 (2016/05/02)
 
 * Use filenames to compute set of files to publish line coverage
+* Fix incorrect line coverage when merging multiple cobertura reports [#167](https://github.com/uber/phabricator-jenkins-plugin/pull/167)
```

Comment:
This should be under the next unreleased version, since 1.9.5 already exists:

```
### 1.9.6 (unreleased)

* Fix incorrect ...
```

No need to reference the PR